### PR TITLE
minor pm parser_updates

### PR DESCRIPTION
**Pull request by claytonpbarrows**
this PR passes kwargs through to parse_file to allow for control on pm_validation routines

### CODE REVIEW

File: src/base.jl [hunk 1]
```diff
@@ -18,6 +18,7 @@ const SYSTEM_KWARGS = Set((
     :time_series_read_only,
     :timeseries_metadata_file,
     :unit_system,
+    :validate,
```
Comment (jd-lara) [inline src/base.jl:21]: How does a user distinguish between validate and run checks? If I set run checks to false and validate to true was to that mean?

**Review by jd-lara — CHANGES_REQUESTED** [review]
the validate keyword needs changing

File: src/base.jl (hunk 1 above)
Comment (jd-lara) [reply src/base.jl:21]: We also need to update the docstring

Comment (claytonpbarrows) [reply src/base.jl:21]: fair... I was just directly passing the pm kwarg... I'll make it more clear

Comment (jd-lara) [reply src/base.jl:21]: FWIW I don't think that validate is correct in PM either since it doesn't validate really but mutates the underlying data.

**Review by jd-lara — APPROVED** [review]

**Review by jd-lara — APPROVED** [review]

**Pull request merged**